src/main/java/haveno/price/ApiController.java

```diff
@@ -0,0 +1,23 @@
+package haveno.price;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.env.Environment;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+public class ApiController {
+
+    private final Environment env;
+
+    @GetMapping("/api/version")
```

Comment:
Wondering if this should be "/api/version" or just "/version" to match "/getAllMarketPrices". Both calls are part of the API, so thinking "/version" makes sense, but I'm open if you have any reasoning toward "/api/version".

Reply:
This is really up to the implementation. When "/api/" appears in a URL, it often indicates that the endpoint being accessed is part of an API, which means it exposes functionality or data that can be accessed programmatically by other software systems rather than being intended for direct human consumption through a web browser.

Reply:
ah.. changed it to /version for the sake of sanity here :D